app/src/androidTest/java/com/mapbox/android/events/testapp/MainActivityInstrumentationTests.java

```diff
@@ -0,0 +1,23 @@
+package com.mapbox.android.events.testapp;
+
+import android.app.Activity;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(AndroidJUnit4.class)
+public class MainActivityInstrumentationTests {
```

Comment:
so what's the point of this test here?

Reply:
Just to run throw the whole process in MainActivity. BTW, this bug was found by this test case, we could add more test cases in future in possible.